test/integration/apiserver/apply/apply_test.go

```diff
@@ -132,6 +133,7 @@ func TestCreateOnApplyFailsWithUID(t *testing.T) {
 		Namespace("default").
 		Resource("pods").
 		Name("test-pod-uid").
+		Param("applier", "apply").
```

Comment:
Seeing how this is used, `applier` seems under-specified and possibly too terse.  As a top level flag on the API, we might want to scope this more such as `applyOwner` or something that stands on its own.